### PR TITLE
1.0

### PR DESCRIPTION
**Pull request by natebird**
- Updated packages to 1.0 versions of Vapor and PostgreSQLDriver
- Minor changes to readme to reflect new source locations.

### CODE REVIEW

File: Sources/VaporPostgreSQL/Provider.swift
Comment (tanner0101) [inline Sources/VaporPostgreSQL/Provider.swift]: If port is not a required arg, it should have a default.

File: Sources/VaporPostgreSQL/Provider.swift
Comment (tanner0101) [inline Sources/VaporPostgreSQL/Provider.swift]: Minor issue, comment formatting.

**Comment by tanner0101** [comment]
After comments, this is good to go. :)

File: Sources/VaporPostgreSQL/Provider.swift
Comment (natebird) [reply Sources/VaporPostgreSQL/Provider.swift]: The main init has the default value. Line 112.

File: Sources/VaporPostgreSQL/Provider.swift
Comment (natebird) [reply Sources/VaporPostgreSQL/Provider.swift]: Fixed.

File: Package.swift
Comment (tanner0101) [inline Package.swift]: Depend on Vapor 1.x

File: Sources/VaporPostgreSQL/Provider.swift
Comment (tanner0101) [reply Sources/VaporPostgreSQL/Provider.swift]: But if you force unwrap here it will crash if no port exists in the config files. You'll need to allow an optional in the init call then use the default port if nil is passed.

**Review by tanner0101 — CHANGES_REQUESTED** [review]

File: Sources/VaporPostgreSQL/Provider.swift
Comment (natebird) [reply Sources/VaporPostgreSQL/Provider.swift]: Ah, ok.

**Review by tanner0101 — APPROVED** [review]

**Comment by natebird** [comment]
@tannernelson Can we override the the codecov check?

**Comment by natebird** [comment]
@tannernelson I guess I don't have write access to this repository either. I can't merge this PR.

**Comment by tanner0101** [comment]
This looks ready to merge. And I've given you write access.

**Comment by natebird** [comment]
Cool. Thanks!

**Pull request merged**
